cogs/afterhours/afterhours.py

```diff
@@ -207,14 +208,42 @@ async def handleChannelDelete(self, channel: discord.abc.GuildChannel):
 
     @commands.group(name="afterhours")
     @commands.guild_only()
-    @checks.mod_or_permissions(manage_messages=True)
     async def afterHours(self, ctx: Context):
         """Configure after-hours exceptions
 
         There's nothing configurable from Discord.
         """
 
-    @afterHours.command(name="set")
+    @checks.mod_or_permissions(manage_messages=True)
+    @afterHours.command(name="setrole")
+    async def afterHoursSetRole(self, ctx: Context, role: discord.Role):
+        """Set the channel for notifications."""
+        await self.config.guild(ctx.guild).get_attr(KEY_ROLE_ID).set(role.id)
+        await ctx.send(f"Set the After Hours role to {role.name}")
+
+    @afterHours.command(name="removerole")
+    async def afterHoursRemoveRole(self, ctx: Context):
+        """Set the channel for notifications."""
+        # check if after hours role is set
+        roleid = await self.config.guild(ctx.guild).get_attr(KEY_ROLE_ID)()
+        if roleid == None:
+            await ctx.send("After Hours role not set, can't remove")
+            return
+        # get after hours role by id
+        role = ctx.guild.get_role(roleid)
+        # if id is no longer valid (role deleted most likely)
+        if role == None:
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) use `is not None` instead.
```suggestion
        if role is None:
```